6-AdvancedScenarios/2-call-api-mt/SPA/src/app/claim-utils.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * Populate claims table with appropriate description
+ * @param {Record} claims ID token claims
+ * @returns claimsTable
+ */
+export const createClaimsTable = (claims: Record<string, string>): any[] => {
+    const claimsTable: any[] = [];
+
+    Object.keys(claims).map((key) => {
```

Comment:
Update method to check for types, [see](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/pull/162/files)

Reply:
done